helperModel/user-model.js

```diff
@@ -0,0 +1,51 @@
+const db = require('../data/dbConfig.js');
+
+module.exports = {
+    addUser,
+    find,
+    findBy,
+    findById,
+    update,
+    remove
+
+}
+
+function find() {
+    return db('users')
+        .select('id', 'username', 'password', 'email');
```

Comment:
The only thing I would leave out is the password, even though it's hashed, but for improved security.